test/smoke/testApps/SpringBootTest/src/smokeTest/java/com/springbootstartertest/smoketest/SpringbootSmokeTest.java

```diff
@@ -24,104 +24,127 @@
 @UseAgent
 public class SpringbootSmokeTest extends AiSmokeTest {
 
-    @Test
-    @TargetUri("/basic/trackEvent")
-    public void trackEvent() throws Exception {
-        mockedIngestion.waitForItems("RequestData", 1);
-        mockedIngestion.waitForItemsInRequest("EventData", 2);
-
-        // TODO get event data envelope and verify value
-        final List<EventData> data = mockedIngestion.getTelemetryDataByTypeInRequest("EventData");
-        assertThat(data, hasItem(new TypeSafeMatcher<EventData>() {
-            final String name = "EventDataTest";
-            Matcher<String> nameMatcher = Matchers.equalTo(name);
-            @Override
-            protected boolean matchesSafely(EventData item) {
+  @Test
+  @TargetUri("/basic/trackEvent")
+  public void trackEvent() throws Exception {
+    mockedIngestion.waitForItems("RequestData", 1);
+    mockedIngestion.waitForItemsInRequest("EventData", 2);
```

Comment:
same here, can you undo formatting changes on this file to make easier to review?